docs/resources/organization.md

```diff
@@ -0,0 +1,29 @@
+# Resource: organization
```

Comment:
I think we should call out in the docs for `organization_settings` that it is deprecated as well

Reply:
And you can also add a `Deprecated` field to the schema of the SDKv2 implementation so it will output on the log when running terraform